src/Pages/Shop/ShopDetail/DetailContent/RightSection/RightSection.js

```diff
@@ -32,15 +36,37 @@ function RightSection({ text }) {
   };
 
   const setOnText = (index) => {
-    setPackage(text.buy[index]);
+    setPackage(text.package_name[index]);
     setIndexPackage(index);
+    setChangeList(index);
   };
 
   const setOnUsers = (index) => {
-    setUserSelect(text.users[index]);
+    setUserSelect(USER_COUNT[index]);
     setIndexUsers(index);
+    setChangeUser(index);
+
+    if (index === 0) {
+      return multiply;
+    } else {
+      setMultiply(index + 1);
+    }
```

Comment:
```suggestion
    if (!index) return multiply;
    if (index) setMultiply(index + 1);
```
한줄 짜리라면 이렇게 써도 될 것 같습니다

Reply:
삼항연사자로 작성해 보았는데 괜찮을까요??!